src/ui/screens/settings/ViewSubscriptionManagement/SubscriptionManagementItem.qml

```diff
@@ -29,7 +29,7 @@ ColumnLayout {
                 visible: false
             }
             PropertyChanges {
-                target: relayUpsell
+                target: upgradeToAnnualSub
```

Comment:
I know you're just following the pattern of the file (and the pattern that is commonly used elsewhere in the app) of adding property changes for each state, but what do you think about removing these, and just adding `visible: type === "text-upgrade"` to the `upgradeToAnnualSub` RowLayout?

Reply:
I'd rather leave it here for now to keep the pattern the same. Agree this file needs a trip to the cleaners though.